readme.md

```diff
@@ -85,14 +85,11 @@ console.log(yellow('foo ' + red().bold('red') + ' bar ' + cyan('cyan') + ' baz')
 Toggle color support as needed; `kleur` includes simple auto-detection which may not cover all cases.
 
 ```js
-const kleur = require('kleur');
+import kleur from 'kleur';
 
-// manually disable
+// disable with boolean result of right side assignment.
 kleur.enabled = false;
 
-// or use another library to detect support
-kleur.enabled = require('color-support').level;
```

Comment:
Why was this removed? This was an important documentation snippet that's been linked to many times. 

If it's just because of the `require` reference, let's leave this entire snippet as a `require`